modules/standard/IO.chpl

```diff
@@ -3308,7 +3308,7 @@ proc fileReader._getFp(): (bool, c_ptr(c_FILE)) {
     return (true, fp);
   } else {
     if qio_get_fd(f, fd) == 0 {
-      fp = fdopen(fd, c"r");
+      fp = fdopen(fd, "r".c_str());
```

Comment:
Am not expecting `.c_str()` is necessary here

Reply:
make check seg faults without it